Labs/Deep Learning/400 - Stream Analytics and Machine Learning/3 - Predict/readme.md

```diff
@@ -228,7 +228,7 @@ In this exercise, you will modify the Azure Function that you created in the pre
 1. Open the Azure Function contained in the Function App. Replace the function code with the following code:
 
 	```javascript
-	module.exports = async function (context, req) {
+	module.exports = function (context, req) {
```

Comment:
An async Azure Function seems to immediately exit upon returning, even if there are still asynchronous callbacks in progress. Minimal reproducible example:

```
module.exports = async function (context, req) {
    setTimeout(() => { context.log('two'); }, 1000);
    context.log('one');
};
```
will output 'one' and never output 'two'. In this case, the function never makes it past sending the request to the Custom Vision Service.

Reply:
Just ran into this this week and am fixing.

Reply:
Thanks!